presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -837,6 +840,28 @@ public void testCreateUnpartitionedTableAsSelect()
         }
     }
 
+    @Test
+    public void testCreateBucketTableAsSelect()
+    {
+        // TODO: enable this after column stats collection is enabled.
+        Session session = Session.builder(getSession())
```

Comment:
This code repeats multiple times. Consider extracting a helper method:

```
Session session tableWrite()
{
   ....
}

assertQueryFails(tableWrite(), "CREATE TABLE...");
```

Reply:
Updated. Thanks!